src/explore-education-statistics-common/src/modules/charts/components/HorizontalBarBlock.tsx

```diff
@@ -81,7 +83,13 @@ const HorizontalBarBlock = ({
   const minorDomainTicks = getMinorAxisDomainTicks(chartData, axes.minor);
   const majorDomainTicks = getMajorAxisDomainTicks(chartData, axes.major);
 
-  const yAxisWidth = parseNumber(axes.major.size);
+  // Enforce a max y axis width on mobile as large widths cause
+  // the chart to not be visible.
+  const maxMobileYAxisWidth = 160;
```

Comment:
Any reason why this particular width has been chosen? Or is it one that just seemed to work?

Reply:
Just from trying different widths to find one that gave enough space to the chart while trying to not cause too many lines of wrapping on long labels.

Reply:
Cool, guess it will do for now!